feature-libs/cart/base/root/models/cart.model.ts

```diff
@@ -81,6 +81,7 @@ export interface Cart {
   purchaseOrderNumber?: string;
   saveTime?: Date;
   savedBy?: Principal;
+  scheduleLines?: ScheduleLine[];
```

Comment:
I think we can use [model augmentation](https://sap.github.io/spartacus-docs/type-augmentation/#augmenting-modules) in this case - just extend the model only in your integration lib.

Reply:
sure, will do that Michal.